nekoyume/Assets/_Scripts/Analyzer.cs

```diff
@@ -0,0 +1,54 @@
+﻿using mixpanel;
+using PackageExtensions.Mixpanel;
+using UnityEngine;
+
+namespace Nekoyume
+{
+    public class Analyzer
+    {
+        public static Analyzer Instance => Game.Game.instance.Analyzer;
+
+        private readonly MixpanelValueFactory _mixpanelValueFactory;
+
+        public Analyzer()
+        {
+            _mixpanelValueFactory = new MixpanelValueFactory();
+        }
+
+        public Analyzer Initialize(string uniqueId = "non-unique-id")
+        {
+#if UNITY_EDITOR
+            Debug.Log("Analyzer does not track in editor mode");
+#else
+            Mixpanel.SetToken("80a1e14b57d050536185c7459d45195a");
```

Comment:
Just out of curiosity, is it okay to expose mixpanel token?

Reply:
@Unengine It is OK. There is mixpanel token in `Game.cs` already. This change is just move it.